src/components/NcRichContenteditable/NcAutoCompleteResult.vue

```diff
@@ -51,7 +51,7 @@ export default {
 	name: 'NcAutoCompleteResult',
 
 	props: {
-		label: {
+		title: {
```

Comment:
This change was intentional, because we agreed to use the term `title` only if if relates to the HTML attribute `title`, which it does not here.
Is there a deeper problem with renaming the prop here?